src-plugins/itkFilters/itkFiltersBinaryCloseProcess_p.h

```diff
@@ -72,8 +72,8 @@ class itkFiltersBinaryCloseProcessPrivate : public itkMorphologicalFiltersProces
         QString newSeriesDescription = input->metadata ( medMetaDataKeys::SeriesDescription.key() );
 
         if (isRadiusInPixels)
-            newSeriesDescription += " Bin.Close filter\n("+ QString::number(radius[0])+", "+
-            QString::number(radius[1])+", "+ QString::number(radius[2])+" pixels)";
+            newSeriesDescription += " Bin.Close filter\n("+ QString::number(floor(radius[0]))+", "+
+            QString::number(floor(radius[1]))+", "+ QString::number(floor(radius[2]))+" pixels)";
```

Comment:
OK, but the context here would be: the user asks for a closing of 5.99 **pixels**. Such a user is an idiot and should be punished. :p 
Jokes aside, I had to choose between rounding and truncating, both are relevant, the simplest was truncating.
But I think this is not an important choice, since it should be a very rare scenario, the metaData says the size of the kernel (top right of the view) so there's no surprise for the user.